docs/_config.yml

```diff
@@ -0,0 +1,39 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: napari
+author: The napari team
+logo: images/logo.png
+# only_build_toc_files: true
+
+# Force re-execution of notebooks on each build.
+# See https://jupyterbook.org/content/execute.html
+execute:
+  execute_notebooks: force
+
+# Information about where the book exists on the web
+repository:
+  url: https://github.com/executablebooks/jupyter-book # Online location of your book
+  path_to_book: docs # Optional path to your book, relative to the repository root
+  branch: master # Which branch of the repository should be used when creating links (optional)
+
+# Add GitHub buttons to your book
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
+
+sphinx:
+  extra_extensions:
+    - sphinx.ext.viewcode
+    - sphinx.ext.napoleon
+    - sphinx.ext.autodoc
+    - sphinx_autodoc_typehints
+    - sphinx.ext.autosummary
+
+  config:
```

Comment:
could consider using a different pygments theme... 

I think the default is a bit drab, 

<img width="633" alt="Untitled2" src="https://user-images.githubusercontent.com/1609449/106402762-1bd6ca80-63f9-11eb-9400-42cbf60b6013.png">

here's `pygments_style: paraiso-dark` ... and you can explore more [here](https://pygments.org/demo/#try).

<img width="632" alt="Untitled" src="https://user-images.githubusercontent.com/1609449/106402763-1bd6ca80-63f9-11eb-88af-243e7d2469ed.png">

Reply:
woaaaaah.  I just realized that this theme will auto-switch light/dark depending on your system settings.  That's pretty bad-ass (I think?)... so might want to keep that in mind before accepting this suggestion (though, I'm still not a huge fan of the yellowish background)

https://user-images.githubusercontent.com/1609449/106403063-a3710900-63fa-11eb-92bc-e1d317ddfd2d.mov

Reply:
That's cool! @liaprins-czi I'm not sure if you've thought about this as you're working on the design. It could be good for accessibility too.

Reply:
Good point, a lot fo apps and websites are allowing to change background themes based on user settings, we could do that too